### PR TITLE
Add bashlex recipe

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**

### CODE REVIEW

File: recipes/bashlex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - enum34
```
Comment (synapticarbors) [inline recipes/bashlex/meta.yaml]: `enum34` shouldn't be a `build` requirement, although I am looking at the `0.12` `setup.py`.

File: recipes/bashlex/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{% set name = "bashlex" %}
+{% set version = "0.11" %}
```
Comment (synapticarbors) [inline recipes/bashlex/meta.yaml]: newest version is `0.12`. Any reason to start with the previous version?

File: recipes/bashlex/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/idank/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (synapticarbors) [inline recipes/bashlex/meta.yaml]: probably a candidate for `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

**Comment by sebastian-luna-valero** [comment]
Done. Many thanks @synapticarbors

File: recipes/bashlex/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - enum34
```
Comment (synapticarbors) [inline recipes/bashlex/meta.yaml]: `enum34` is not built for py3.6:

https://github.com/conda-forge/enum34-feedstock/blob/master/recipe/meta.yaml#L16

so you'll need a selector tag on this requirement, which also means that you can't use `noarch: python`.

File: recipes/bashlex/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - bashlex
+
+about:
+  home: https://github.com/idank/bashlex
+  license: GPL-3.0
+  license_family: GPL
```
Comment (synapticarbors) [inline recipes/bashlex/meta.yaml]: Please manually include the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and open an issue upstream about including it in the source distribution. If you could also put a copy of the issue url in the `meta.yaml` as a comment that would be great. This way there is a reminder to check on the status of the issue next time the feedstock is updated.

**Comment by sebastian-luna-valero** [comment]
@conda-forge/staged-recipes  please review

**Pull request merged**